vnext/Desktop.IntegrationTests/DesktopTestRunner.cpp

```diff
@@ -72,7 +73,11 @@ shared_ptr<ITestInstance> TestRunner::GetInstance(
       std::make_tuple(
           TestDeviceInfoModule::name,
           []() -> unique_ptr<CxxModule> { return std::make_unique<TestDeviceInfoModule>(); },
-          nativeQueue)};
+          nativeQueue),
+
+      {TestDevSettingsModule::name,
+       []() -> unique_ptr<CxxModule> { return std::make_unique<TestDevSettingsModule>(); },
```

Comment:
nit:indentation seems off as well as curly is not consistent with the first value of the tuple.

Reply:
should it use `sd:make_tuple`?

Reply:
it does, albeit implicitly ;)

That said, I shall update the rest of the declarations to match the style.

Reply:
I have no preference for whichever style :) I just like consistency :D